src/js/rebuses.js

```diff
@@ -9,6 +9,11 @@ const rebuses = [
     words: ['Houseplants'],
     hint: `The second emoji is 'ant' not 'bug'`
   },
+  {
+    symbols: ['(after ❌)', '+(🐝-b)','+☕️'],
```

Comment:
Add a space before the last item in the array to make the linter happy. Also, could you move this down to the end of the array? 🙂